clash-ghc/src-ghc/Clash/GHC/LoadModules.hs

```diff
@@ -304,10 +306,12 @@ loadModules useColor hdl modName dflagsM idirs = do
     topEntities' <-
       case (topEntities, topSyn) of
         ([], []) ->
-          Panic.pgmError $ unwords [ "No 'topEntity', nor function with a"
-                                   , "'Synthesize' annotation found in root"
-                                   , "module:"
-                                   , (Outputable.showSDocUnsafe (ppr rootModule)) ]
+          let modName1 = Outputable.showSDocUnsafe (ppr rootModule) in
+          Panic.pgmError [I.i|
+            No top-level function called 'topEntity' found, nor a function with
+            a 'Synthesize' annotation in module #{modName1}. Did you forget to
+            export them?
```

Comment:
Maybe suggest looking at `Clash.Annotations.TopEntity` for more details? New users will have no idea what a synthesize annotation is

Reply:
Good idea.

Reply:
Done